portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,31 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
+
+
 public class DataServlet extends HttpServlet {
+  
+  private ArrayList<String> list;
+
+  public DataServlet(){ 
+    list = new ArrayList<String>();
```

Comment:
It's better to use "this" inside a java class for accessing private attributes. So this will be "this.list".

Reply:
Also, you can initialize arraylist with values. it's cleaner (I think).

So `list = new ArrayList<String>(){"Fruit", "Soap", "Cereal"};`

Reply:
When trying to initialize arraylist that way, I'm getting an error that says "illegal start of type".

Reply:
I think that newer initializer might not be available in Java 8? Or maybe that's C++?

Reply:
Ha, my java skill is rusty now. 

You can change change to 
```
private List<String> list;

list = Arrays.asList("Fruit", "Soap", "Cereal"};
```

Also, I just noticed that it's better to declare list with interface instead of implementation. List is an interface, and ArrayList is one of the implementations of List. It can also has other implementations such LinkedList. By declaring a variable with interface, it's easier to switch implementations without having to modify clients' code.